xla/debug_options_flags.cc

```diff
@@ -1266,6 +1268,13 @@ void MakeDebugOptionsFlags(std::vector<tsl::Flag>* flag_list,
                     &DebugOptions::set_xla_gpu_enable_nccl_clique_optimization),
                 debug_options->xla_gpu_enable_nccl_clique_optimization(),
                 "Allow early return when acquiring NCCL cliques"));
+  flag_list->push_back(tsl::Flag(
+      "xla_gpu_enable_while_loop_double_buffering",
+      bool_setter_for(
+          &DebugOptions::set_xla_gpu_enable_while_loop_double_buffering),
+      debug_options->xla_gpu_enable_while_loop_double_buffering(),
+      "Enable double buffering for while loop"));
+
```

Comment:
The internal linter complains about this blank line :-)

Reply:
Some context: Linter errors will not prevent merging. We can fix them as part of merging PRs.

That said, merging is a little easier for us if no linter errors exist, so removing that blank line is still appreciated :-)